src/user-story/jenkins-is-the-way-for-networkupstools/index.yaml

```diff
@@ -0,0 +1,124 @@
+      The library also helps determine which C or C++ language revisions can be
+      built according to GCC or CLANG version installed. Whenever a build
+      starts, the NUT Jenkinsfile (consumer of jenkins-dynamatrix library) can
+      evaluate what it can build today -- based on currently known population of
+      build agents, and so constructs the matrix for testing.
+    - This arrangement allows NUT to be built on a multitude of platforms such
+      that common free CI platforms do not offer. It is possible to find ways to
+      build FOSS on recent Linux, MacOS and Windows, and sometimes even on
+      non-x86 CPUs. It is nearly impossible to find builders for FreeBSD,
+      OpenBSD or Solaris/illumos, to name a few, or on Linux distributions older
+      than half a dozen years.
+    - Every iteration of NUT is now tested with GNU, BSD and Sun Make
+      implementations, with Shell helpers tested by Bash, Dash, Ksh, Busybox and
+      others, with GCC starting from early 4.x releases and Clang from 3.x up to
+      the dozenth releases out recently, across a dozen hardware platforms (some
+      in QEMU), and several revisions of C/C++ standard with and without GNU
+      extensions. All in all, with recently regularly connected builders about
+      150 stages are prepared (some running several build scenarios as scripted
+      inside, so roughly a thousand builds happen). Sometimes a nuance warning
+      causes one of these scenarios to complain during PR builds -- on this or
+      that OS, this or that compiler. It keeps us confident both that such
+      mechanism remains useful, and that any OS the community cares about enough
+      to contribute a builder for remains fully supported.
+    - With the hundreds of builds succeeding without warnings -- reliably,
+      repeatably, every time -- the floodgates for ingestion of long-standing
+      PRs were open. Largest beasts included the dual-citizenship with
+      libusb-0.1 and libusb-1.x support (queued for 5 years) and revival of NUT
+      for Windows effort (queued for 9 years), confidently merged without
+      introducing warnings and regressions. The overwhelming majority of these
+      builds happen on the custom "NUT CI" farm with Jenkins driving them. A few
+      scenarios run on CircleCI and Appveyor to take advantage of some free FOSS
+      builds on MacOS and Windows respectively.
+    - Of notable interactions with the Jenkins community, this and related
+      efforts led to improvements in the git-client-plugin to extend the use of
+      reference repositories, instant-messaging-plugin and ircbot-plugin for
+      querying Jenkins server status via IRC servers, lockable-resources plugin
+      for many itches scratched here and there, and pipeline-groovy-lib-plugin
+      for easier use of test library versions along with test pipeline branches.
+      And very many lessons learned and friends made :)
+---
```

Comment:
```suggestion
```

You have multiple yaml documents in this file (--- is a divider of documents), you need to remove the second one.

Reply:
Well, that's what CMS generated and submitted... Thanks!